docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -28,11 +28,31 @@ This page explains how to get started monitoring the security posture of your cl
 [[cspm-setup]]
 == Set up CSPM for AWS
 
-You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access.
+You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access. Two deployment technologies are available: agentless, and agent-based. <<cspm-aws-agentless, Agentless deployment>> allows you to collect cloud posture data without having to manage the deployment of an agent in your cloud. <<cspm-aws-agent-based, Agent-based deployment>> requires you to deploy and manage an agent in the cloud account you want to monitor. 
```

Comment:
Not sure if you're specifically referring to Elastic's Agent here or agents in a general sense. If it's the former, this change applies across all related pages. 

```suggestion
You can set up CSPM for AWS either by enrolling a single cloud account, or by enrolling an organization containing multiple accounts. Either way, first you will add the CSPM integration, then enable cloud account access. Two deployment technologies are available: agentless, and agent-based. <<cspm-aws-agentless, Agentless deployment>> allows you to collect cloud posture data without having to manage the deployment of {agent} in your cloud. <<cspm-aws-agent-based, Agent-based deployment>> requires you to deploy and manage {agent} in the cloud account you want to monitor. 
```